provisioning/install-precice.sh

```diff
@@ -14,10 +14,10 @@ fi
     cd precice
     git pull
     mkdir -p build && cd build/
-    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=OFF -Wno-dev ..
+    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=OFF -Wno-dev ..
     make -j "$(nproc)"
     rm -fv ./*.deb && make package
-    sudo apt-get install -y ./libprecice2_*.deb
+    sudo apt-get install -y ./libprecice3_*.deb
```

Comment:
Any reason not to keep this version agnostic?

```suggestion
    sudo apt-get install -y ./libprecice*_*.deb
```